R/import_mzroll.R

```diff
@@ -168,6 +168,110 @@ process_mzroll <- function(mzroll_db_path,
   return(mzroll_list)
 }
 
+#' Process Metabolon
+#' 
+#' Import results generated by metabolon into romic structure, using series of CSV
+#' output files delivered by Metabolon as inputs.
+#' 
+#' @param peak_areas_path path to peak areas CSV file from metabolon results
+#' @param chem_anno_path path to chemical annotations CSV file from metabolon results
+#' @param metadata_path path to metadata CSV file from metabolon results
+#' @param peak_floor_intensity value to use to replace NA peak measurements (missing values)
+#' 
```

Comment:
its nice to have examples for exported functions. Is there a simple published metabolon dataset that you might be able to use? (read.csv / read_csv take urls as a path)?